README.md

```diff
@@ -6,59 +6,82 @@
 [![codecov](https://codecov.io/gh/nitrictech/nitric/branch/develop/graph/badge.svg?token=20TYFIQS2P)](https://codecov.io/gh/nitrictech/nitric)
 [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=nitrictech_membrane&metric=alert_status)](https://sonarcloud.io/dashboard?id=nitrictech_membrane)
 
-[Nitric](https://nitric.io) is a provider independent framework and runtime for cloud-native and serverless applications. Using Nitric, applications take advantage of cloud-native services for events, queues, compute, APIs, storage, and documents without direct integration to cloud specific APIs.
+[Nitric](https://nitric.io) is a framework for rapid development of cloud-native and serverless applications. Define your apps in terms of the resources they need, then write the code for serverless function based APIs, event subscribers and scheduled jobs.
 
-This decoupling enables applications to remain portable between cloud-providers and alternate deployment options such as Kubernetes or stand-alone servers, from a single application codebase.
+Apps built with Nitric can be deployed to AWS, Azure or Google Cloud all from the same code base so you can focus on your products, not your cloud provider.
+
+Nitric makes it easy to:
+
+- Create smart serverless functions and APIs
+- Build reliable distributed apps that use events and/or queues
+- Securely store, retrieve and rotate secrets
+- Read and write files from buckets
 
 ## Documentation
 
-The full documentation is available at [nitric.io/docs](https://nitric.io/docs)
+The full documentation is available at [nitric.io/docs](https://nitric.io/docs).
+
+We also encourage [contributions](https://github.com/nitrictech/nitric/discussions).
```

Comment:
Does this imply monetary contributions? Should this be "We're completely open source, and encourage contributions or discussion"

Reply:
Since this is for code contributions, maybe link to the contributions page on the website https://nitric.io/docs/contributions

Reply:
Changed both the description to say open source and encourage code contributions.
Updated the link.